packages/sanity/src/core/create/studio-app/useStudioAppIdStore.ts

```diff
@@ -15,14 +16,12 @@ export interface ResolvedStudioApp {
  *
  * @internal
  */
-export function useStudioAppIdStore(
-  cache: AppIdCache,
-  config: {
-    enabled: boolean
-    fallbackStudioOrigin?: string
-  },
-): ResolvedStudioApp {
+export function useStudioAppIdStore(config: {
+  enabled: boolean
+  fallbackStudioOrigin?: string
+}): ResolvedStudioApp {
   const client = useClient({apiVersion: '2024-09-01'})
+  const cache = useAppIdCache()
```

Comment:
Forgive me, I might be missing something, would _just_ keeping the cache here (while not being used anywhere) just set it up?

Reply:
I presume so since we also do that in the `SanityCreateConfigProvider` but